internal/databases/mongo/binary/dto.go

```diff
@@ -2,28 +2,42 @@ package binary
 
 import (
 	"os"
+	"path"
+	"time"
 )
 
-type BackupDirectoryMeta struct {
-	Path     string      `json:"Path"`
-	FileMode os.FileMode `json:"FileMode"`
+type MongodBackupFilesMetadata struct {
+	BackupDirectories []*BackupFileMeta `json:"BackupDirectories,omitempty"` // used only for restoring old backups
+	BackupFiles       []*BackupFileMeta `json:"BackupFiles,omitempty"`
 }
 
-type Checksum struct {
-	Algorithm string `json:"Algorithm"`
-	Data      string `json:"Data"`
+type BackupFileMeta struct {
+	Path        string      `json:"Path"`
+	FileMode    os.FileMode `json:"FileMode"`
+	Compression string      `json:"Compression"` // used only for restoring old backups
+	FileSize    int64       `json:"-"`
```

Comment:
is it intended?

Reply:
yes, I want skip it in json, but it used during backup process.